### PR TITLE
Updates following Noise sync changes

### PR DESCRIPTION
**Pull request by lucafavatella**

### CODE REVIEW

File: epoch/api/README.md [hunk 1]
```diff
@@ -3,26 +3,29 @@
...
 ## Overview
 
-The epoch node API consists of the following endpoints:
-* External HTTP API;
-  * It consists of the peer-to-peer network API and of part of the user API;
-  * It is defined via Swagger schema;
+The epoch node exposes the following APIs:
+* Peer-to-peer network API. It consists of one TCP endpoint:
+  * It is encrypted and authenticated using the Noise protocol;
+  * It is not defined in a document. Advanced users may inspect the `aec_peer_connection` module;
```
Comment (hanssv) [inline epoch/api/README.md]: not yet? (we intend to describe, just not in this intermediate state)

Comment (lucafavatella) [reply epoch/api/README.md]: Addressed.

File: epoch/api/README.md [hunk 2]
```diff
@@ -3,26 +3,29 @@
...
+  * Internal HTTP endpoint;
+    * It is defined via Swagger schema;
+    * It is **not** meant to be exposed on the Internet;
+    * Its TCP port is configurable.
+  * Internal WebSocket endpoint.
+    * It is defined in the rest of this document;
+    * It is **not** meant to be exposed on the Internet;
+    * Its TCP port is configurable.
 
 The epoch node uses the plain HTTP protocol for its API endpoints during the testnet phase while it will use the HTTPS protocol in mainnet.
```
Comment (lucafavatella) [inline epoch/api/README.md]: @sennui @velzevur Any comments on this like welcome.

Comment (dincho) [reply epoch/api/README.md]: This HTTPS concerns me a bit, because users have to deploy their own certs to enable it.

**Pull request merged**